core/src/jvmMain/kotlin/de/p7s1/qa/sevenfacette/db/Database.kt

```diff
@@ -85,6 +87,34 @@ class Database(
         return result
     }
 
+    /**
+     * Executes a prepared statement until time duration is reached and returns a JSONArray.
+     *
+     * @param [preparedDbStatement] statement to execute
+     *
+     * @return [JSONArray]
+     */
+    fun waitUntilExistsOrUpdated(preparedDbStatement: SqlStatement, pollingTime: Duration) : JSONArray? {
```

Comment:
please rename pollingTime to timeout (which is kind of default for test tools) and give it a default value